request-manager/grapher/graph.go

```diff
@@ -27,13 +27,16 @@ type Graph struct {
 // should be retried on error. Next defines all the out edges
 // from Node, and Prev defines all the in edges to Node.
 type Node struct {
-	Datum     job.Job                // Data stored at this Node
-	Next      map[string]*Node       // out edges ( node id -> Node )
-	Prev      map[string]*Node       // in edges ( node id -> Node )
-	Name      string                 // the name of the node
-	Args      map[string]interface{} // the args the node was created with
-	Retry     uint                   // the number of times to retry a node
-	RetryWait uint                   // the time, in seconds, to sleep between retries
+	Datum           job.Job                // Data stored at this Node
+	Next            map[string]*Node       // out edges ( node id -> Node )
+	Prev            map[string]*Node       // in edges ( node id -> Node )
+	Name            string                 // the name of the node
+	Args            map[string]interface{} // the args the node was created with
+	Retry           uint                   // the number of times to retry a node
+	RetryWait       uint                   // the time, in seconds, to sleep between retries
+	SequenceStartId string
+	SequenceEndId   string
+	SequenceRetry   uint // only recorded on start node
```

Comment:
I don't understand the code comment. What's "recorded"? And what does "on start node" mean--oh, the first node in the sequence?

Reply:
yeah the first node in the sequence. i'll update